Sources/Runner/Commands/Runner.swift

```diff
@@ -122,12 +122,16 @@ func runDanger(logger: Logger) throws {
     // swiftlint:enable line_length
 
     var args = [String]()
-    args += ["--driver-mode=swift"] // Eval in swift mode, I think?
     args += libArgs
     args += [tempDangerfilePath] // The Dangerfile
     args += Array(CommandLine.arguments.dropFirst()) // Arguments sent to Danger
-    args += [dslJSONPath] // The DSL for a Dangerfile from DangerJS
-    args += [dangerResponsePath] // The expected for a Dangerfile from DangerJS
+
+    let fileArgs = [dslJSONPath, dangerResponsePath] // The DSL for a Dangerfile from DangerJS and The expected for a Dangerfile from DangerJS
```

Comment:
<!--
  0 failure: 
  1 warning:  Line should be 12...
  
  
  DangerID: danger-id-default;
  File: Sources/Runner/Commands/Runner.swift;
  Line: 129;
-->

- :warning: Line should be 120 characters or less: currently 142 characters (`line_length`)